openassessment/templates/openassessmentblock/staff_debug/student_info.html

```diff
@@ -66,6 +66,34 @@ <h4 class="title--sub">{% trans "Overall Feedback" %}</h4>
         </div>
             {% endif %}
 
+        {% if problem_closed or scores.points_earned > -1 %}
+        <div class="staff-info__status ui-staff__content__section">
+            <h2 class="title">{% trans "Scores" %}</h2>
+            <table class="staff-info__status__table">
+                <thead>
+                <tr>
+                    <th abbr="Peer Score" scope="col">{% trans "Peer Score" %}</th>
```

Comment:
> The abbr attribute has no visual effect in ordinary web browsers, but can be used by screen readers. [1]

This turns out to be user-facing, so we should be translating, but...

> The abbr attribute specifies a shorter version of the content in a header cell. [1]

So if we're not actually abbreviating anything, we should remove this attribute entirely.

[1] http://www.w3schools.com/tags/att_th_abbr.asp